src/Microsoft.AspNet.JsonPatch/JsonPatchDocument.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.JsonPatch.Adapters;
+using Microsoft.AspNet.JsonPatch.Converters;
+using Microsoft.AspNet.JsonPatch.Exceptions;
+using Microsoft.AspNet.JsonPatch.Helpers;
+using Microsoft.AspNet.JsonPatch.Operations;
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNet.JsonPatch
+{
```

Comment:
Missing documentation. Can you take it relevant parts from `JsonPatchDocumentOfT` class and add the rest?